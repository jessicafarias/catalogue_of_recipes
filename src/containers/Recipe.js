const Recipe = () => (
  <div>
    {/* Display the recipe of the selected meal */}
  </div>
);

export default Recipe;
