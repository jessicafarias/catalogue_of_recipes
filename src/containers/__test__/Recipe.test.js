import { BrowserRouter } from 'react-router-dom';
import React from 'react';
import '@testing-library/jest-dom';
import TestRenderer from 'react-test-renderer'; // ES6
import Recipe from '../Recipe';

describe('Display Recipe.js', () => {
  it('renders correctly', () => {
    const tree = TestRenderer.create(<BrowserRouter><Recipe /></BrowserRouter>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
