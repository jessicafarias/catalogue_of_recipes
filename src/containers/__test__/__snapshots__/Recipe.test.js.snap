// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Display Recipe.js renders correctly 1`] = `<div />`;
